network/test/src/lib.rs

```diff
@@ -561,11 +561,13 @@ pub trait TestNetFactory: Sized {
 		);
 		let verifier = VerifierAdapter::new(Arc::new(Mutex::new(Box::new(verifier) as Box<_>)));
 
+		let spawner = |future| spawn_task_handle.spawn_blocking("import-queue-worker", future);
```

Comment:
I'm not sure how you expect that to work. There's ` spawn_task_handle` here.

Reply:
Yeah I know haha I'm having a hard time managing to build Polkadot locally cuz I'm getting some "currently not active?!" cargo metada panic, so I'm currently kind of coding in the dark :p